src/app/leverage/components/leverage-widget/components/summary.tsx

```diff
@@ -125,7 +132,7 @@ export function Summary() {
                       ? formatDollarAmount(Number(inputTokenAmountUsd) * 0.001)
                       : ''}
                   </div>
-                </div>
+                </div> */}
```

Comment:
only commenting out tooltip in case we decide to bring it back soon (was unclear)